src/discussions/comments/data/slices.js

```diff
@@ -52,6 +52,8 @@ const commentsSlice = createSlice({
     totalPages: null,
     totalThreads: null,
     postStatus: RequestStatus.SUCCESSFUL,
+    courseState: RequestStatus.SUCCESSFUL,
+    courseSettings: null,
```

Comment:
I don't think this belongs under comments. It should have its own data files.

Reply:
Also this should have a proper structure so you avoid the ?.
```suggestion
    courseSettings: {
        divisionScheme: null
    },
```

Reply:
> I don't think this belongs under comments. It should have its own data files.

What do you mean by "its own data file"? Where would the appropriate "file" be located?

Reply:
What I mean is that course settings shouldn't be a part of comments. It should be hosted in `src/discussions/data/*` as we'd discussed in out meeting.